requirements-testing.txt

```diff
@@ -0,0 +1,6 @@
+boto3
+black[jupyter]
+plotly
+pyqtgraph
+git+https://github.com/ConservationInternational/trends.earth-schemas.git@v2.1.2
```

Comment:
This is a requirement of the plugin itself (not just of the tests), so it would be better to manage this through the regular plugin dependencies than separately here (otherwise this is just another place to keep the correct version number in sync with that specified in the regular requirements.txt file). [This function](https://github.com/ConservationInternational/trends.earth/blob/main/tasks.py#L658) should be used to install those dependencies when the plugin is being setup on the server for the CI tests.